### PR TITLE
Time entry report updates

### PR DESCRIPTION
**Pull request by onkar-saeloun**
## Notion card
https://www.notion.so/saeloun/Modifications-to-Time-Entry-Report-a2c8491934f14b6cb5313dac723693f8

## Summary

Added updates to time entry report UI

## Preview

https://user-images.githubusercontent.com/104751221/179936993-b4c2b684-fd9e-4834-b1f4-5f19378924d6.mp4

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Tested Manually on my local machine.

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/javascript/src/components/Reports/Container/index.tsx [hunk 1]
```diff
@@ -35,9 +36,14 @@ const Container = () => {
 
   return (
     <Fragment>
+      {false && <TotalHeaderSingleAmt firstTitle={"TOTAL LOGGED HOURS"} firstAmount={"120:45"} />}
       {
         timeEntryReport.reports.map((report, index) => (
           <Fragment key={index}>
+            {false && <div className="flex justify-between border-b border-miru-han-purple-1000 pt-5 pb-2 items-center mt-3">
+              <h1 className="text-miru-han-purple-1000 font-bold text-xl ">{report.label}</h1>
+              <p>Total Hours {report.label && `for ${report.label}`} : 66:09</p>
```
Comment (JijoBose) [inline app/javascript/src/components/Reports/Container/index.tsx:45]: Are we hard-coding for now?

**Review by JijoBose — APPROVED** [review]

**Comment by apoorv1316** [comment]
@onkar Total hours implementation is missing on this PR. Also, the UI is different from the card.

**Comment by onkar-saeloun** [comment]
> @onkar Total hours implementation is missing on this PR. Also, the UI is different from the card.

@apoorv1316 currently the API integration is not done yet. As we don't have any values to show I have hided it. It will appear after API integration. let me know if you want to know how to enable it on local machine.

**Comment by apoorv1316** [comment]
> > @onkar Total hours implementation is missing on this PR. Also, the UI is different from the card.
> 
> @apoorv1316 currently the API integration is not done yet. As we don't have any values to show I have hided it. It will appear after API integration. let me know if you want to know how to enable it on local machine.

@onkar-saeloun API integration will be done in a different PR? If yes then can we merge this?

**Comment by onkar-saeloun** [comment]
> > > @onkar Total hours implementation is missing on this PR. Also, the UI is different from the card.
> > 
> > 
> > @apoorv1316 currently the API integration is not done yet. As we don't have any values to show I have hided it. It will appear after API integration. let me know if you want to know how to enable it on local machine.
> 
> @onkar-saeloun API integration will be done in a different PR? If yes then can we merge this?

Yes @apoorv1316 API integration will be done in separate PR

**Review by apoorv1316 — APPROVED** [review]

**Comment by apoorv1316** [comment]
@supriya3105 are we holding off this PR?

**Comment by supriya3105** [comment]
@onkar-saeloun The backend changes for this requirement has been merged. Once you are done with charter , you can continue work on this PR.

**Comment by onkar-saeloun** [comment]
> @onkar-saeloun The backend changes for this requirement has been merged. Once you are done with charter , you can continue work on this PR.

Okay

**Comment by supriya3105** [comment]
Closing it . Onkar will another PR once work is finished on this PR>

**Pull request closed without merging**